src/components/table/user/user-table.tsx

```diff
@@ -3,6 +3,7 @@ import { QueryClient, QueryClientProvider, useQuery } from "react-query"
 
 import { DataTable } from "../data-table"
 import { Columns } from "./columns"
+import UserRole from "@/../types/enums"
 
 const UserTable = () => {
   const queryClient = new QueryClient()
```

Comment:
can help to remove `QueryClientProvider` from this page and from `points.tsx` and wrap it in `_app.tsx` instead, multiple providers will cause problems for react query 
```suggestion
    <QueryClientProvider client={queryClient}>
        <Component {...pageProps} />
    <QueryClientProvider client={queryClient}>
```

Reply:
Alright, made the changes at https://github.com/cs301-itsa-ay2324/admin-ui-proxy/pull/5/commits/68211eaacd0eb7e5fe1469ef93fbdf286d980a4a